frontend/src/components/Like/Like.css

```diff
@@ -0,0 +1,17 @@
+.like {
+    margin-left: 10px;
+}
+
+.like button {
+    display: block;
+    height: 100%;
+    background-color: #a7c957;
+    color: white;
+    border: none;
+    border-radius: 4px;
+    cursor: pointer;
+}
+
+.like button:hover {
+    background-color: #6a994e;
+}
```

Comment:
not a priority, but probably could play around with some css to avoid the buttons shifting when the text shrinks/expands  from Like -> Liked or Dislike -> Disliked in the video